libportal/portal-helpers.h

```diff
@@ -23,11 +23,11 @@
 
 G_BEGIN_DECLS
 
-#ifndef __GTK_DOC_IGNORE__
+#ifndef __DOCS_IGNORE__
```

Comment:
I renamed `gtk_doc` to `docs` but I am not sure if this change was correct or where is this variable coming from.